src/style.css

```diff
@@ -0,0 +1,62 @@
+
+body {
+  margin: 0;
+}
+
+.stars_block {
+  margin: auto;
+  width: auto;
+  height: auto;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+  
+ 
+
+.stars .stars__star {
+  background-image: url(../../../layout_stars/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+    
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../../../layout_stars/src/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(../../../layout_stars/src/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(../../../layout_stars/src/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../../../layout_stars/src/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../../../layout_stars/src/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
You repeat the same logic here. Just combine these selectors, please 
<img width="1110" alt="image" src="https://user-images.githubusercontent.com/99403198/219003283-b40d7ae0-6212-4c60-b6f4-894d9d5eee62.png">